### DISCUSSION TITLE
Getting :"Redis ERROR {\"command\":\"INFO\",\"code\":\"DENIED\"}","v":1}"

### DISCUSSION THREAD

**Discussion by munagalabhanuprakash** [learndevopsonline/learndevopsonline#1525] [Q&A]
Hi Raghu,

I am trying to deploy roboshop project on centos 7 AMI
As per the follwoing step : https://lucky-hide-390.notion.site/04-Redis-606886ca6cab43ba9c7149c88246e15a
I tried to install redis 6.2.11 on on my linux VM, But it has thrown an error as following
![image](https://github.com/learndevopsonline/learndevopsonline/assets/104569251/b7f518b4-43dc-4e33-880b-f5f06d7bf26f)

So i have installed the latest by using the command **yum install redis**
i have also changed the bind ip 0.0.0.0 in the config file
followed the other steps. Redis is up and running and listening on  6379, PFA
![image](https://github.com/learndevopsonline/learndevopsonline/assets/104569251/b2b3c036-9dba-48cc-8629-7877567ebeb2)

Now i have proceeded with Users and cart applications and performd the steps as per the notion doc
and when i checked the status of user and the cart application i am seeing the following errors
![image](https://github.com/learndevopsonline/learndevopsonline/assets/104569251/f14d510e-9eda-4f56-b992-ea71217a7fc0)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/104569251/5056b395-8f29-4eb5-9e62-d200917108f4)

i have  checked the listen port of redis and changed tried by changing the bind ip to bind * -::*. but nothing has. changed
Looks like a connection denied error.

Please help here

FYI:

  | mongodb | i-011105a291d276a59 | 54.87.192.164 | 172.31.21.198
-- | -- | -- | -- | --
  | catalogue | i-0a2e99dd2d1a545d0 | 54.235.47.15 | 172.31.16.58
  | user | i-083f4e2a4c5382725 | 54.91.14.30 | 172.31.26.228
  | cart | i-06ff90f0a0ef31383 | 52.91.141.75 | 172.31.31.52
  | redis | i-0e543670cf183b2d1 | 54.144.72.20 | 172.31.29.208
  | frontend | i-0f00353f0c85d92ec | 107.20.82.253 | 172.31.27.228

**Comment by sreedharm07** [comment]
Hi @munagalabhanuprakash 
in  https://lucky-hide-390.notion.site/04-Redis-606886ca6cab43ba9c7149c88246e15a  redis version was updated,  please try now to install redis as per in doc, it will install

**Comment by munagalabhanuprakash** [comment]
It Worked @sreedharm07 

Thanks
Bhanu
